src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -197,12 +195,6 @@ def check_user
     User.session = User.find_by_login(session[:login]) if session[:login]
 
     User.session ||= User.possibly_nobody
-
-    if !User.session
```

Comment:
@hennevogel why did you remove this `if/else` block? doesn't seem to be related to the proxy mode

Reply:
You are right, this code would also run without proxy mode. But it still would generate "wrong" data.

Once `User.session` is false we run an unauthenticated request, but that isn't data we want to track. And `previous_user` being different from `User.session` does *not* mean a log in happened. 

Let me split those to another commit...

Reply:
Did not find a way to split this more intelligent so I made a new commit message. Please check again.

Reply:
okay understand :+1: